app/views/users/_form.html.erb

```diff
@@ -69,12 +69,28 @@
     <div class='tab-pane' id='roles'>
       <%= checkbox_f f, :admin if User.current.can_change_admin_flag? %>
       <%= multiple_checkboxes f, :roles, @user, Role.givable.for_current_user,
-                               { :label => _('Roles')}, {:disabled => @editing_self ? Role.givable.for_current_user.pluck(:id) : false } %>
+                              { :label => _('Roles')}, {:disabled => @editing_self ? Role.givable.for_current_user.pluck(:id) : false } %>
+      <div class="form-group">
+        <% if @user.cached_usergroups.uniq.any? %>
+            <label class="col-md-2 control-label" for="roles">Roles inherited from user groups</label>
```

Comment:
Nitpick, if you can shorten it to `Roles from user groups` that'd be good, it looks kind of odd now
![screenshot from 2016-08-08 08-06-46](https://cloud.githubusercontent.com/assets/598891/17470762/35f1c188-5d3f-11e6-8361-0447aa7c1414.png)
![screenshot from 2016-08-08 08-07-24](https://cloud.githubusercontent.com/assets/598891/17470763/35f31506-5d3f-11e6-9d53-43acc7febac1.png)